en_US/advanced/http-api.md

```diff
@@ -1811,7 +1811,7 @@ The license contents to be uploaded.
 Upload a license file:
 
 ```sh
-$ curl -XPOST --basic -u admin:public -d @<(jq -sR < path/to/new.license) 'http://localhost:8081/api/v4/license/upload'
+$ curl -XPOST --basic -u admin:public -d @<(jq -sR '{license: .}' < path/to/new.license) 'http://localhost:8081/api/v4/license/upload'
```

Comment:
I wonder why it appears in the logs that the POSTed data was double escaped. (e.g. `\\n`)
not entirely sure if it happened here by the curl command, or if it was the log from EMQX

Does this produce the same result ?
```
jq -sR '{license: .}' < path/to/new.license > data.json
curl -XPOST --basic -u admin:public -d @./data.json 'http://localhost:8081/api/v4/license/upload'
```

Reply:
Strange, I tested locally with the branch and it worked.  :thinking: 

```sh
ͳ curl -sXPOST --basic -u admin:public -d @<(jq -sR '{license: .}' < data/emqx.lic) 'http://localhost:8081/api/v4/license/upload' | jq '.'
{
  "message": "ok",
  "code": 200
}

ͳ jq -sR '{license: .}' < data/emqx.lic > data.json
ͳ curl -sXPOST -u admin:public -d @./data.json 'http://localhost:8081/api/v4/license/upload' | jq '.'
{
  "message": "ok",
  "code": 200
}
```

And I saw no logs in EMQX.